package.json

```diff
@@ -31,19 +31,19 @@
   },
   "homepage": "https://github.com/digicatapult/dscp-ipfs#readme",
   "dependencies": {
-    "@digicatapult/dscp-node": "^4.1.0",
+    "@digicatapult/dscp-node": "^4.2.1",
     "dotenv": "^8.2.0",
     "envalid": "^7.0.0",
     "express": "^4.17.1",
     "pino": "^6.11.1",
     "pino-http": "^5.5.0"
   },
   "devDependencies": {
-    "babel-eslint": "^10.1.0",
+    "@babel/eslint-parser": "^7.18.9",
```

Comment:
we can maybe remove babel? If change `eslintrc` to not use it as a parser

Reply:
Apparently we actually need babel for this repo